api_tests/src/actors/actor.ts

```diff
@@ -337,7 +338,7 @@ export class Actor {
     /**
      * Takes a BtcDai sell order (herc20-hbit Swap)
      */
-    public async takeOrder() {
+    public async takeOrder(partialTakeAmount: string) {
```

Comment:
I think this parameter should be optional, otherwise one has to unnecessarily pass in the full amount again if you want to take the full order.